host_core/lib/host_core.ex

```diff
@@ -147,14 +167,6 @@ defmodule HostCore do
 
   defp write_config(config) do
     write_json(config, "./host_config.json")
-
-    case System.user_home() do
-      nil ->
-        Logger.warn("Can't check for ~/.wash host config - no user home available")
-
-      h ->
-        write_json(config, Path.join([h, "/.wash/", "host_config.json"]))
-    end
```

Comment:
I thought where we landed was that we shouldn't be _reading_ from `~/.wash/host_config.json` but we should still _write_ that file as that's what `wash ctx` uses for connection values

Reply:
:thinking: ok. this effectively means that wash will always, by default, use the same settings as the most recently started wasmCloud host on that machine.

Reply:
Correct, that was my goal thinking of `wash` and the use of contexts primarily a developer tool. You can always "turn this off" by selecting your own default context and using those values